Gemfile.lock

```diff
@@ -1,202 +1,241 @@
   specs:
-    health-data-standards (3.6.1)
-      activesupport (~> 4.1.1)
+    health-data-standards (3.7.0)
+      activesupport (~> 4.2.0)
       builder (~> 3.1)
       erubis (~> 2.7.0)
       highline (~> 1.7.0)
       log4r (~> 1.1.10)
       memoist (~> 0.9.1)
-      mongoid (~> 4.0.0)
-      mongoid-tree (~> 1.0.4)
-      nokogiri (~> 1.6.7.1)
+      mongoid (~> 5.0.0)
+      mongoid-tree (~> 2.0.0)
+      nokogiri (~> 1.8.2)
       protected_attributes (~> 1.0.5)
       rest-client (~> 1.8.0)
-      rubyzip (= 0.9.9)
+      rubyzip (~> 1.2.1)
       uuid (~> 2.3.7)
+      zip-zip (~> 0.3)
 
 GIT
   remote: https://github.com/projecttacoma/patientapi.git
-  revision: 8f6a7657cacb0ef27b6a495ba4e5d636d1edf694
-  branch: master
+  revision: 4caa8d0fa8d796c8e8469adc5b58919d12fce117
+  branch: bonnie-prior_1302_nokogiri_fix
   specs:
     hquery-patient-api (1.0.4)
 
 GEM
-  remote: http://rubygems.org/
+  remote: https://rubygems.org/
   specs:
-    actionmailer (4.1.12)
-      actionpack (= 4.1.12)
-      actionview (= 4.1.12)
+    actionmailer (4.2.10)
+      actionpack (= 4.2.10)
+      actionview (= 4.2.10)
+      activejob (= 4.2.10)
       mail (~> 2.5, >= 2.5.4)
-    actionpack (4.1.12)
-      actionview (= 4.1.12)
-      activesupport (= 4.1.12)
-      rack (~> 1.5.2)
+      rails-dom-testing (~> 1.0, >= 1.0.5)
+    actionpack (4.2.10)
+      actionview (= 4.2.10)
+      activesupport (= 4.2.10)
+      rack (~> 1.6)
       rack-test (~> 0.6.2)
-    actionview (4.1.12)
-      activesupport (= 4.1.12)
+      rails-dom-testing (~> 1.0, >= 1.0.5)
+      rails-html-sanitizer (~> 1.0, >= 1.0.2)
+    actionview (4.2.10)
+      activesupport (= 4.2.10)
       builder (~> 3.1)
       erubis (~> 2.7.0)
-    activemodel (4.1.12)
-      activesupport (= 4.1.12)
+      rails-dom-testing (~> 1.0, >= 1.0.5)
+      rails-html-sanitizer (~> 1.0, >= 1.0.3)
+    activejob (4.2.10)
+      activesupport (= 4.2.10)
+      globalid (>= 0.3.0)
+    activemodel (4.2.10)
+      activesupport (= 4.2.10)
       builder (~> 3.1)
-    activerecord (4.1.12)
-      activemodel (= 4.1.12)
-      activesupport (= 4.1.12)
-      arel (~> 5.0.0)
-    activesupport (4.1.12)
-      i18n (~> 0.6, >= 0.6.9)
-      json (~> 1.7, >= 1.7.7)
+    activerecord (4.2.10)
+      activemodel (= 4.2.10)
+      activesupport (= 4.2.10)
+      arel (~> 6.0)
+    activesupport (4.2.10)
+      i18n (~> 0.7)
       minitest (~> 5.1)
-      thread_safe (~> 0.1)
+      thread_safe (~> 0.3, >= 0.3.4)
       tzinfo (~> 1.1)
     ansi (1.5.0)
-    arel (5.0.1.20140414130214)
-    awesome_print (1.6.1)
-    bson (3.2.6)
-    builder (3.2.2)
-    coderay (1.1.0)
-    coffee-rails (4.1.0)
+    arel (6.0.4)
+    awesome_print (1.8.0)
+    bson (4.3.0)
+    builder (3.2.3)
+    bundler-audit (0.6.0)
+      bundler (~> 1.2)
+      thor (~> 0.18)
+    coderay (1.1.2)
+    coffee-rails (4.2.2)
       coffee-script (>= 2.2.0)
-      railties (>= 4.0.0, < 5.0)
+      railties (>= 4.0.0)
```

Comment:
coffee-rails currently has the following dependency on railties, so I think this should be `railties (>= 4.0.0, <5.1)`:
![image](https://user-images.githubusercontent.com/14879344/37927048-f62a7606-3106-11e8-84b9-be517326ff2f.png)

Reply:
@losborne and I looked at this and reproduced it on his machine; for some reason bundler doesn't add this restriction to coffee-rails dependencies. This removal wasn't hand-jammed. Not going to mess with it.